pylib/cqlshlib/cql3handling.py

```diff
@@ -1501,6 +1507,38 @@ def alter_type_field_completer(ctxt, cass):
                        ;
 '''
 
+syntax_rules += r'''
+<slaName> ::= <identifier>
+             | <quotedName>
+             | <unreservedKeyword>
+             ;
+
+<createSlaStatement> ::= "CREATE" "SERVICE_LEVEL" ( "IF" "NOT" "EXISTS" )? <slaName>
+                              ( "WITH" <slaProperty> ("AND" <slaProperty>)*)?
+                        ;
+
+<alterSlaStatement> ::= "ALTER" "SERVICE_LEVEL" ("IF" "EXISTS")? <slaName>
+                              ( "WITH" <slaProperty> ("AND" <slaProperty>)*)?
+                       ;
+
+<slaProperty> ::= "WORKLOAD_TYPE" "=" <stringLiteral>
+                 | "TIMEOUT" "=" <wholenumber>
```

Comment:
where are those documented? On https://enterprise.docs.scylladb.com/stable/using-scylla/workload-prioritization.html I only see `SHARES`

Reply:
@Lorak-mmk You can find it [here](https://opensource.docs.scylladb.com/stable/using-scylla/workload-attributes.html)

Reply:
it's a part in OSS part in enterprise